### PR TITLE
Feature/wc scanner decription text

### LINKED ISSUE #246: Description text is missing in the WC scanner
According to Figma, a descriptive text should be displayed below the square to focus the scanner as seen in the image below:

https://www.figma.com/design/uyqKWH9xX5hOz0BjtuzJ7g/Qubic-Wallet-Mobile-App?node-id=2113-1934&t=QjqKBdcfeiOLpXMo-4
image

Suggestion: I would like to have it of the frame instead, let's see how it fits there.

Please use the following text in English:
English: "Open a Web3 app that supports WalletConnect. Scan the app's QR code, or paste its connection link."
Spanish: "Abre una aplicación Web3 que sea compatible con WalletConnect. Escanea el código QR de la app o pega su enlace de conexión."
German: "Öffne eine Web3-App, die WalletConnect unterstützt. Scanne den QR-Code der App oder füge den Verbindungslink ein."
French: "Ouvrez une application Web3 qui prend en charge WalletConnect. Scannez le code QR de l'application ou collez son lien de connexion."
Russian: "Откройте Web3-приложение, поддерживающее WalletConnect. Сканируйте QR-код приложения или вставьте его ссылку для подключения."
Turkish: "WalletConnect'i destekleyen bir Web3 uygulamasını açın. Uygulamanın QR kodunu tarayın veya bağlantı linkini yapıştırın."
Chinese Traditional: "打開支援 WalletConnect 的 Web3 應用程式。掃描應用程式的 QR 碼，或貼上其連接鏈接。"

### PR DESCRIPTION
**Pull request by ahmed-tarek-salem**
Closes #246

### CODE REVIEW

File: lib/pages/main/wallet_contents/add_wallet_connect/components/add_wallet_connect_mobile_view.dart [hunk 1]
```diff
@@ -96,6 +96,25 @@ class _AddWalletConnectMobileViewState
                   ),
                 ),
               ),
+            // Text below the scan window
```
Comment (sallymoc) [inline lib/pages/main/wallet_contents/add_wallet_connect/components/add_wallet_connect_mobile_view.dart:99]: @ahmed-tarek-salem  please move up in the screen the frame to scan and the instructions

Comment (ahmed-tarek-salem) [reply lib/pages/main/wallet_contents/add_wallet_connect/components/add_wallet_connect_mobile_view.dart:105]: Fixed with b75a593, tested on large and small mobile screens.

File: lib/styles/responsive_constants.dart [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class ResponsiveConstants {
+  static const double largeScreenHeight = 700;
+  static const double mediumScreenHeight = 500;
```
Comment (sallymoc) [inline lib/styles/responsive_constants.dart:3]: @ahmed-tarek-salem  where is mediumScreenHeight being used?

Comment (ahmed-tarek-salem) [reply lib/styles/responsive_constants.dart]: @sallymoc 
I used it first and then removed it from the code, resolved.

**Pull request merged**